include/onnxruntime/core/framework/allocator.h

```diff
@@ -39,6 +39,9 @@ constexpr const char* CUDA = "Cuda";
 constexpr const char* CUDA_PINNED = "CudaPinned";
 constexpr const char* MIGRAPHX = "MIGraphX";
 constexpr const char* MIGRAPHX_PINNED = "MIGraphXPinned";
+constexpr const char* OpenVINO = "OpenVINO";
```

Comment:
where is OpenVINO string getting used?

Reply:
To create Default CPU Allocator for OpenVINO in openvino_execution_provider.cc
AllocatorCreationInfo device_info(
      [](OrtDevice::DeviceId) {
        return CreateCPUAllocator(OrtMemoryInfo(OpenVINO, OrtDeviceAllocator));
      });

Reply:
should we make it OpenVINO_CPU and OpenVINO_GPU then?

Reply:
Changed to OpenVINO_CPU